graph.go

```diff
@@ -294,12 +303,18 @@ func (g *Graph) PruneBelowDepth(targetDepth uint64) int {
 	}
 
 	for id, depth := range g.missing {
-		if depth > targetDepth {
-			continue
+		if depth <= targetDepth {
+			delete(g.missing, id)
+			count++
```

Comment:
Care to put count++ after the two delete() statements?

Reply:
not sure about children, since its just reference to transaction's children, not another transaction